ts/src/apex.ts

```diff
@@ -0,0 +1,1940 @@
+        //             "digitMerge": "0.1,0.2,0.4,1,2",
+        //             "displayMaxLeverage": "100",
+        //             "displayMinLeverage": "1",
+        //             "enableDisplay": true,
+        //             "enableOpenPosition": true,
+        //             "enableTrade": true,
+        //             "fundingImpactMarginNotional": "6",
+        //             "fundingInterestRate": "0.0003",
+        //             "incrementalInitialMarginRate": "0.00250",
+        //             "incrementalMaintenanceMarginRate": "0.00100",
+        //             "incrementalPositionValue": "50000.0000",
+        //             "initialMarginRate": "0.01",
+        //             "maintenanceMarginRate": "0.005",
+        //             "maxOrderSize": "50",
+        //             "maxPositionSize": "100",
+        //             "minOrderSize": "0.0010",
+        //             "maxMarketPriceRange": "0.025",
+        //             "settleAssetId": "USDT",
+        //             "baseTokenId": "BTC",
+        //             "stepSize": "0.001",
+        //             "symbol": "BTC-USDT",
+        //             "symbolDisplayName": "BTCUSDT",
+        //             "tickSize": "0.1",
+        //             "maxMaintenanceMarginRate": "0.5000",
+        //             "maxPositionValue": "5000000.0000",
+        //             "tagIconUrl": "https://static-pro.apex.exchange/icon/LABLE_HOT.svg",
+        //             "tag": "HOT",
+        //             "riskTip": false,
+        //             "defaultInitialMarginRate": "0.05",
+        //             "klineStartTime": 0,
+        //             "maxMarketSizeBuffer": "0.98",
+        //             "enableFundingSettlement": true,
+        //             "indexPriceDecimals": 2,
+        //             "indexPriceVarRate": "0.001",
+        //             "openPositionOiLimitRate": "0.05",
+        //             "fundingMaxRate": "0.000234",
+        //             "fundingMinRate": "-0.000234",
+        //             "fundingMaxValue": "",
+        //             "enableFundingMxValue": true,
+        //             "l2PairId": "50001",
+        //             "settleTimeStamp": 0,
+        //             "isPrelaunch": false,
+        //             "riskLimitConfig": {},
+        //             "category": "L1"
+        //         }
+        //     ]
+        // }
+        return this.parseMarkets (perpetualContract);
+    }
+
+    parseMarket (market: Dict): Market {
+        const marketId = this.safeString (market, 'symbol');
+        const quoteId = this.safeString (market, 'l2PairId');
+        const baseId = this.safeString (market, 'baseTokenId');
+        const quote = this.safeString (market, 'settleAssetId');
+        const base = this.safeCurrencyCode (baseId);
+        const settleId = this.safeString (market, 'settleAssetId');
+        const settle = this.safeCurrencyCode (settleId);
+        const symbol = this.safeString (market, 'crossSymbolName');
```

Comment:
we should parse it as the unified symbol, you can read more about it here: 

https://docs.ccxt.com/#/README?id=symbols-and-market-ids